### PR TITLE
AO3 4472 Home page rounded counts

### PR DESCRIPTION
**Pull request by zz9pzza**
https://otwarchive.atlassian.net/browse/AO3-4472

Round the counts down. It is still possible to to flick between two numbers ( One in squid and one in memcached however it is less likely ). This isn't a bug it is to do with squid not caching perfectly as we don't cache the first page before we get the right cookies in the browser.

### CODE REVIEW

**Comment by sarken** [comment]
Since it's not longer just one short line per item, I suspect we might want to put the rounded counts in /decorators/homepage.rb, with everything else for the homepage

**Comment by zz9pzza** [comment]
@sarken done

File: app/decorators/homepage.rb [hunk 1]
```diff
@@ -3,6 +3,19 @@ def initialize(user)
...
 
+  def rounded_counts
+    @user_count = Rails.cache.fetch("/v1/home/counts/user", expires_in: 40.minutes) do
+      estimate_number(User.count)
+    end
+    @work_count = Rails.cache.fetch("/v1/home/counts/works", expires_in: 40.minutes) do
+      estimate_number(Work.posted.count)
+    end
+    @fandom_count = Rails.cache.fetch("/v1/home/counts/fandom", expires_in: 40.minutes) do
+      estimate_number(Work.posted.count)
```
Comment (sarken) [inline app/decorators/homepage.rb]: This is fandoms, so I think it should be `Fandom.canonical.count`

**Comment by zz9pzza** [comment]
Sorry about that sarken :(

**Comment by sarken** [comment]
Okay, I think we're good! :+1:

**Pull request merged**